diff -Naur udisks-2.1.4/src/udiskslinuxfilesystem.c udisks-2.1.4.tpg/src/udiskslinuxfilesystem.c
--- udisks-2.1.4/src/udiskslinuxfilesystem.c	2014-12-18 11:00:34.000000000 +0000
+++ udisks-2.1.4.tpg/src/udiskslinuxfilesystem.c	2015-01-20 18:30:37.697476606 +0000
@@ -894,42 +894,10 @@
   /* If we know the user-name and it doesn't have any '/' character in
    * it, mount in /run/media/$USER
    */
-  if (!fs_shared && (user_name != NULL && strstr (user_name, "/") == NULL))
-    {
-      mount_dir = g_strdup_printf ("/run/media/%s", user_name);
-      if (!g_file_test (mount_dir, G_FILE_TEST_EXISTS))
-        {
-          /* First ensure that /run/media exists */
-          if (g_mkdir ("/run/media", 0755) != 0 && errno != EEXIST)
-            {
-              g_set_error (error,
-                           UDISKS_ERROR,
-                           UDISKS_ERROR_FAILED,
-                           "Error creating directory /run/media: %m");
-              goto out;
-            }
-          /* Then create the per-user /run/media/$USER */
-          if (g_mkdir (mount_dir, 0700) != 0 && errno != EEXIST)
-            {
-              g_set_error (error,
-                           UDISKS_ERROR,
-                           UDISKS_ERROR_FAILED,
-                           "Error creating directory `%s': %m",
-                           mount_dir);
-              goto out;
-            }
-          /* Finally, add the read+execute ACL for $USER */
-          if (!add_acl (mount_dir, uid, error))
-            {
-              if (rmdir (mount_dir) != 0)
-                udisks_warning ("Error calling rmdir() on %s: %m", mount_dir);
-              goto out;
-            }
-        }
-    }
   /* otherwise fall back to mounting in /media */
-  if (mount_dir == NULL)
-    mount_dir = g_strdup ("/media");
+  // Remove code
+  //if (mount_dir == NULL)
+  mount_dir = g_strdup ("/media");
 
   /* NOTE: UTF-8 has the nice property that valid UTF-8 strings only contains
    *       the byte 0x2F if it's for the '/' character (U+002F SOLIDUS).
