diff -Naur udisks-2.1.4/configure.ac udisks-2.1.4.tpg/configure.ac
--- udisks-2.1.4/configure.ac	2014-12-18 09:35:11.000000000 +0000
+++ udisks-2.1.4.tpg/configure.ac	2015-01-20 18:12:41.926381735 +0000
@@ -126,7 +126,7 @@
 AC_SUBST(GLIB_CFLAGS)
 AC_SUBST(GLIB_LIBS)
 
-PKG_CHECK_MODULES(GIO, [gio-unix-2.0 >= 2.36])
+PKG_CHECK_MODULES(GIO, [gio-unix-2.0 >= 2.36 gmodule-2.0])
 AC_SUBST(GIO_CFLAGS)
 AC_SUBST(GIO_LIBS)
 
